FWCore/Framework/src/processBlockUtilities.cc

```diff
@@ -0,0 +1,11 @@
+#include "FWCore/Framework/interface/processBlockUtilities.h"
+
+#include "FWCore/Framework/interface/Event.h"
+
+namespace edm {
+
+  unsigned int eventProcessBlockIndex(Event const& event, std::string const& processName) {
+    return event.processBlockIndex(processName);
```

Comment:
I'm wondering the value of this function. Is it to avoid including `Event.h` in places that need this functionality?

Reply:
Yes. Without it, Event.h gets included in every module. A lot of them include it already, but I don't think all unless I missed something.